src/dex/balancer-v2/balancer-v2-integration.test.ts

```diff
@@ -59,9 +66,9 @@ describe('BalancerV2', function () {
         pools,
       );
       console.log('WETH <> DAI Pool Prices: ', poolPrices);
-
       expect(poolPrices).not.toBeNull();
       checkPoolPrices(poolPrices!, amounts, SwapSide.SELL, dexKey);
+      expect(poolPrices?.[0].gasCost).toBe(25); // TO DO
```

Comment:
Currently failing but will be updated when gasCost is finalized.

Reply:
for now I just added some constant based on the simulation, can be improved later